easybuild/easyconfigs/b/Blender/Blender-3.3.1.eb

```
@@ -0,0 +1,32 @@
+easyblock = 'PackedBinary'
+
+name = 'Blender'
+version = '3.3.1'
+versionsuffix = ''
+
+homepage = 'https://www.blender.org/'
+description = """Blender is the free and open source 3D creation suite. It supports
+ the entirety of the 3D pipeline-modeling, rigging, animation, simulation, rendering,
+ compositing and motion tracking, even video editing and game creation."""
+
+toolchain = {'name': 'system', 'version': ''}
```

```suggestion
toolchain = SYSTEM
```

See also [system toolchain specs](https://docs.easybuild.io/en/latest/System_toolchain.html).